apps/commudle-admin/src/app/feature-modules/events/components/event-locations/event-location-tracks/event-location-tracks.component.scss

```diff
@@ -17,14 +17,17 @@
     }
 
     nb-accordion-item-header {
-      @apply com-py-1 com-px-2;
+      @apply com-py-1 com-px-2 com-overflow-x-auto com-justify-between;
       background-color: #ffaa00;
       @apply com-border-0 com-rounded-md;
```

Comment:
let's make this as a single line and convert background color to tailwind css

Reply:
Done